democracy_club/templates/projects.html

```diff
@@ -11,18 +11,16 @@ <h1>Projects</h1>
 
     <div class="feature-card-wrapper">
 
-      <article class="feature-card" itemscope itemtype="http://schema.org/WebSite" id="project--wheredoivote">
-        <h3 class="feature-card__title" itemprop="name"><a href="https://wheredoivote.co.uk/" itemprop="url" target="_blank">Where Do I Vote? <i class="fa fa-external-link feature-card__icon"></i></a></h3>
+      <article class="feature-card">
+        <h3 class="feature-card__title"><a href="{% url "polling_one_pager" %}">
```

Comment:
I think it would be useful for our website to have a link to https://wheredoivote.co.uk/ somewhere on it. Also, the way we've done all the other things on this page is that the header links to _the actual thing_ e.g:

https://whocanivotefor.co.uk/
https://candidates.democracyclub.org.uk/
https://elections.democracyclub.org.uk/

etc

I reckon the section for WhereDIV should do the same, or we should change all the others. I know you were saying you want to re-design the 'projects' page a bit. Maybe its better to do that bit in another PR?

Reply:
If anything I want to remove the links on the others – I find that page _really_ confusing because there are 100 buttons and links and I have no idea what any of them do without thinking really hard. Every. Damn. Time.